bin/configs/scala-legacy.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: scala-httpclient-deprecated
+outputDir: samples/client/petstore/scala-httpclient-deprecated
+inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
+templateDir: modules/openapi-generator/src/main/resources/scala-httpclient
+additionalProperties:
+  apiPackage: org.openapitools.example.api
+  invokerPackage: org.openapitools.example.invoker
+  artifactId: scala-legacy-petstore
```

Comment:
@sandeepjindal  Please run and commit those files
```
./bin/generate-samples.sh bin/configs/scala-legacy.yaml
```

Reply:
Done please check